async-query/src/main/java/org/opensearch/sql/spark/flint/OpenSearchFlintIndexClient.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.spark.flint;
+
+import lombok.RequiredArgsConstructor;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.action.admin.indices.delete.DeleteIndexRequest;
+import org.opensearch.action.support.master.AcknowledgedResponse;
+import org.opensearch.client.Client;
+
+@RequiredArgsConstructor
+public class OpenSearchFlintIndexClient implements FlintIndexClient {
+  private static final Logger LOG = LogManager.getLogger();
+
+  private final Client client;
+
+  @Override
+  public void deleteIndex(String indexName) {
+    LOG.info("Vacuuming Flint index {}", indexName);
```

Comment:
Shouldnt this log line be in original Operation file.

Reply:
Fixed.